independent-projects/arc/runtime/src/main/java/io/quarkus/arc/impl/ComputingCache.java

```diff
@@ -80,18 +80,24 @@ public void clear() {
 
     public void forEachValue(Consumer<? super V> action) {
         Objects.requireNonNull(action);
-        for (LazyValue<V> value : map.values()) {
-            action.accept(value.get());
-        }
+        map.forEach(new BiConsumer<K, LazyValue<V>>() {
+            @Override
+            public void accept(K k, LazyValue<V> vLazyValue) {
+                action.accept(vLazyValue.get());
+            }
+        });
```

Comment:
Given they all do the same thing in this class, wouldn't be a good idea to have a static nested class + a static instance?

Reply:
Sure yeah

Reply:
Actually no as you need to access `action`

Reply:
Ah yes, stupid me.